eclipse-che/che-theia
# fix: pin to the same lerna version in devDeps as in resolutions, so we can use package.json for offline brew builds

### What does this PR do?

fix: pin to the same lerna version in devDeps as in resolutions, so we can use package.json for offline brew builds; also update yarn.lock to remove some of the errors

Change-Id: I3b6b72893ff3396d871f3a630831fa01e52882bf
Signed-off-by: nickboldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-2600

Followup tio https://github.com/eclipse-che/che-theia/commit/86effe8bbe572dcd076641d65fdc91f748c668bd

### How to test this PR?

Before this PR:

```
$ yarn check

yarn check v1.22.5
warning Lockfile has incorrect entry for "lerna@^4.0.0". Ignoring it.
...
error Found 2791 errors.
```

After this PR:

```
$ yarn check
...
info Found 28 warnings.
error Found 181 errors.

```
### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
